cmd/skopeo/utils.go

```diff
@@ -47,6 +48,11 @@ func sharedImageFlags() ([]cli.Flag, *sharedImageOptions) {
 			Usage:       "path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
 			Destination: &opts.authFilePath,
 		},
+		cli.StringFlag{
+			Name:        "big-files-temporary-dir",
```

Comment:
Can we just call this --tmpdir?

Reply:
I would love this to be for the whole tmpdir, but it might lead to confusion: I believe I saw some parts of containers/image or skopeo that used ioutil.TempDir directly without overriding with the big-files-temporary-dir system context value for where to put the directory. I would assume ioutil.TempDir does the right thing by default and honours TMPDIR/TMP environment variable(s).

Reply:
Most (but not all) of the `ioutil.Temp*` calls create a temporary file in some specific directory, to implement atomic replacement.

> I would assume ioutil.TempDir does the right thing by default and honours TMPDIR/TMP environment variable(s).

On Unix, yes, it does (falling back to `/tmp`).

(Overall, I don’t feel strongly about the name at all.)